backends/p4tools/modules/testgen/targets/bmv2/test_backend/metadata.cpp

```diff
@@ -19,62 +19,12 @@
 #include "nlohmann/json.hpp"
 
 #include "backends/p4tools/modules/testgen/lib/test_object.h"
-#include "backends/p4tools/modules/testgen/lib/tf.h"
 #include "backends/p4tools/modules/testgen/targets/bmv2/test_spec.h"
 
 namespace P4Tools::P4Testgen::Bmv2 {
 
-Metadata::Metadata(std::filesystem::path basePath, std::optional<unsigned int> seed = std::nullopt)
-    : TF(std::move(basePath), seed) {}
-
-std::vector<std::pair<size_t, size_t>> Metadata::getIgnoreMasks(const IR::Constant *mask) {
```

Comment:
I didn't see where this got moved to? Maybe it's just removed?

Reply:
Yes, turns out this code was not used at all.

Reply:
Haha, ok.